_docs/tasks/traffic-management-v1alpha3/ingress.md

```diff
@@ -177,35 +166,97 @@ the `istio-ingressgateway` service.
    x-envoy-upstream-service-time: 48
    ```
 
-   ```command
-   $ curl -I -k https://$SECURE_INGRESS_HOST/status/200
-   HTTP/1.1 200 OK
-   server: envoy
-   date: Mon, 29 Jan 2018 04:45:49 GMT
-   content-type: text/html; charset=utf-8
-   access-control-allow-origin: *
-   access-control-allow-credentials: true
-   content-length: 0
-   x-envoy-upstream-service-time: 96
-   ```
-
 1. Access any other URL that has not been explicitly exposed. You should
-   see a HTTP 404 error:
+   see an HTTP 404 error:
 
    ```command
-   $ curl -I http://$INGRESS_HOST/headers
+   $ curl --resolve foo.bar.com:$INGRESS_PORT:$INGRESS_HOST -I http://foo.bar.com:$INGRESS_PORT/headers
    HTTP/1.1 404 Not Found
    date: Mon, 29 Jan 2018 04:45:49 GMT
    server: envoy
    content-length: 0
    ```
 
+### Add a secure port (HTTPS) to our gateway
+
+In this subsection we add to our gateway the port 443 to handle the HTTPS traffic. We create a secret with a certificate and a private key. Then we replace the previous `Gateway` definition with a definition that contains a server on the port 443, in addition to the previously defined server on the port 80.
```

Comment:
Then we update the previous Gateway definition by adding a new ..
There is a bit of grammatical stuff to fix here as well. Will let @frankbu handle the wording.